### DISCUSSION TITLE
Just wanted to say thanks for an awesome project!

### DISCUSSION THREAD

**Discussion by theidealist** [russhughes/s3lcd#33] [General]
I know that large open source contributions like this don't ever get the recognition they deserve, and I just wanted to offer my heartfelt thanks for an awesome project that enables my hacking hobby!  The clarity of the code examples, the excellence of the documentation (including a lot of font styles!) and the README, are all way above the norm.  

I'm using the wt32sc01+ with micropython and this is __exactly__ what I was looking for!  Thank you!

**Comment by russhughes** [comment]
You're very welcome.
